acceptance/acceptance_test.go

```diff
@@ -44,6 +45,7 @@ const (
 	EntryPointScript = "script"
 	CleanupScript    = "script.cleanup"
 	PrepareScript    = "script.prepare"
+	MaxFileSize      = 100_000
```

Comment:
Any particular reason for 100kb limit?

Reply:
It seemed like a reasonable limit on output file. It's conservative - we unlikely to hit it. For context, largest output.txt we have today is acceptance/help/output.txt - 12KB.